bots/task/__init__.py

```diff
@@ -359,7 +359,7 @@ def branch(context, message, pathspec=".", issue=None, **kwargs):
 
     return "{0}:{1}".format(user, branch)
 
-def pull(branch, body=None, issue=None, **kwargs):
+def pull(branch, body=None, issue=None, labels=['bot'], **kwargs):
```

Comment:
Compound default arguments are a bit of a trap (due to passing by reference, not by value), but as it's nowhere written to/changed, it seems okay.